accounts/accounts.go

```diff
@@ -1,10 +1,36 @@
 package accounts
 
-import "errors"
+import (
+	"errors"
+	"time"
+
+	"github.com/distributeddesigns/currency"
+	"github.com/op/go-logging"
+
+)
+
+var (
+	log = logging.MustGetLogger("audit")
+)
+
+type BuyAction struct {
+	time	time.Time
+	stock	string
+	units	uint
+}
+
+type SellAction struct {
+	time	time.Time
+	stock	string
+	units	uint
+}
 
 // Account : State of a particular account
 type Account struct {
-	Balance float64
+	Balance	currency.Currency
+	BuyQueue []BuyAction
+	SellQueue []SellAction
```

Comment:
Will `BuyAction` and `SellAction` be different at some point? This could be `BuyQueue, SellQueue []Action`